setup.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2022.
+# (C) Copyright IBM 2022, 2023.
```

Comment:
Not quite sure why this was updated just for the date - maybe the date tool did this

Reply:
I had done some other changes which I reverted later. This can be reverted, too :+1:
```suggestion
# (C) Copyright IBM 2022.
```